.github/workflows/generate-changelog.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     if: github.repository == 'pinterest/ktlint'
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4
```

Comment:
We can replace these hash strings with `v4`, instead of annoying updates by Renovate.

@paul-dingemans

Reply:
According to Github best practices, the safest approach is to use the SHA commit id instead of version tags. See https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions

Reply:
Yeah, but the following updates will be a little frequent.

Reply:
Ok, I wil see what happens. If it annoys me too much, I can always remove the SHA's and fallback to the version numbers.